cmd/autoowners/main.go

```diff
@@ -259,7 +261,7 @@ func pullOwners(gc github.Client, configRootDir string, blocklist blocklist, con
 
 	for _, orgRepo := range orgRepos {
 		logrus.WithField("orgRepo", orgRepo.repoString()).Info("handling repo ...")
-		httpResult, err := getOwnersHTTP(gc, orgRepo)
+		httpResult, err := getOwnersHTTP(gc, orgRepo, pc.OwnersFilenames(orgRepo.Organization, orgRepo.Repository))
```

Comment:
`pc.OwnersFilenames(orgRepo.Organization, orgRepo.Repository)` hopefully never returns us things like empty strings.

Reply:
No it doesn't: https://github.com/openshift/ci-tools/blob/75fa6a905caa4d0b49ce73c08fa544263c9fbab8/vendor/k8s.io/test-infra/prow/plugins/config.go#L188-L197